Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-aks-cluster.md

```diff
@@ -32,19 +32,30 @@ Using the same PowerShell session from the previous step, update the state file
 Once the files are updated, deploy using Terraform Init, Plan and Apply.
 
 ```PowerShell
-terraform init -input=false -backend-config="resource_group_name=$backendResourceGroupName" -backend-config="storage_account_name=$backendStorageAccountName" -backend-config="container_name=$backendContainername" -backend-config="key=$layerNametfstate" -backend-config="subscription_id=$ARM_SUBSCRIPTION_ID" -backend-config="tenant_id=$tenantId" -backend-config="client_id=$servicePrincipalId" -backend-config="client_secret=$servicePrincipalKey"
+terraform init -input=false -backend-config="resource_group_name=$backendResourceGroupName" -backend-config="storage_account_name=$backendStorageAccountName" -backend-config="container_name=$backendContainername" -backend-config="key=$layerNametfstate"
 ```
 
 ```PowerShell
-terraform plan -out $layerNametfstate -input=false -var="subscription_id=$ARM_SUBSCRIPTION_ID" -var="tenant_id=$tenantId" -var="client_id=$servicePrincipalId" -var="client_secret=$servicePrincipalKey" -var="resource_group_name=$backendResourceGroupName" -var="storage_account_name=$backendStorageAccountName" -var="container_name=$backendContainername" -var="access_key=$access_key" -var="state_sa_name=$backendStorageAccountName" 
+terraform plan -out $layerNametfstate -input=false -var="resource_group_name=$backendResourceGroupName" -var="storage_account_name=$backendStorageAccountName" -var="container_name=$backendContainername" -var="access_key=$access_key" -var="state_sa_name=$backendStorageAccountName" 
 ```
 
 ```PowerShell
-terraform apply -var="subscription_id=$ARM_SUBSCRIPTION_ID" -var="tenant_id=$tenantId" -var="client_id=$servicePrincipalId" -var="client_secret=$servicePrincipalKey" -var="resource_group_name=$backendResourceGroupName" -var="storage_account_name=$backendStorageAccountName" -var="container_name=$backendContainername" -var="access_key=$access_key" 
+terraform apply -var="resource_group_name=$backendResourceGroupName" -var="storage_account_name=$backendStorageAccountName" -var="container_name=$backendContainername" -var="access_key=$access_key" 
 ```
 
 If you get an error about changes to the configuration, go with the `-reconfigure` flag option.
 
+### Exercise the User Nodepool
+The reference implementation creates 3 nodepools: a Linux system nodepool, a Linux user nodepool and a Windows nodepool. The Linux user nodepool is not actively used in this implementation, but the intended uses for this nodepool include hosting ingress controllers, 
+
+Try out the following command to exercise this nodepool
+```
+az aks command invoke \
+  --resource-group $AKS_RESOURCE_GROUP \
+  --name $AKS_CLUSTER_NAME \
+  --command "kubectl get pods -n kube-system"
+```
```

Comment:
~~~suggestion
### Try the user nodepool

The reference implementation creates three nodepools: a Linux system nodepool, a Windows nodepool, and a Linux user nodepool. The Linux user nodepool is not actively used in this implementation, but the intended uses for this nodepool include hosting ingress controllers, or any other workload in the cluster that requires a Linux host -- which is common when running Windows clusters.

This example uses AKS command invoke, which creates a temporary pod to invoke a command on the cluster.  This common management command creates a transient pod in the cluster, and that pod requires a Linux node pool to be available.

```PowerShell
az aks command invoke \
  --resource-group $AKS_RESOURCE_GROUP \
  --name $AKS_CLUSTER_NAME \
  --command "kubectl get pods -n kube-system"
```
~~~